.cargo/config

```diff
@@ -1,8 +1,2 @@
 [alias]
 xtask = "run -p xtask --"
-
```

Comment:
moved these into `examples/.cargo/config` because rustflags in `.cargo/config` compose and w/o the move I ran into knurling-rs/probe-run#143